src/equiv.js

```diff
@@ -59,6 +59,10 @@ QUnit.equiv = (function() {
 		return false;
 	}
 
+	function getRegExpFlags( regexp ) {
+		return regexp.flags || regexp.toString().match(/[gimuy]*$/)[0];
```

Comment:
The CI coverage is fine, I just want to see additional tests added to [test/main/deepEqual.js](https://github.com/mixed/qunit/blob/enhanceCompareRegexpOption/test/main/deepEqual.js#L447) that include regular expressions with the unicode flag set (skipped in environments that don't support it).